[2024-03-03T03:37:54.349+0000] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: fetch_data_files.get_files_2 scheduled__2016-08-22T00:00:00+00:00 [queued]>
[2024-03-03T03:37:54.370+0000] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: fetch_data_files.get_files_2 scheduled__2016-08-22T00:00:00+00:00 [queued]>
[2024-03-03T03:37:54.371+0000] {taskinstance.py:2193} INFO - Starting attempt 1 of 1
[2024-03-03T03:37:54.414+0000] {taskinstance.py:2214} INFO - Executing <Task(PythonOperator): get_files_2> on 2016-08-22 00:00:00+00:00
[2024-03-03T03:37:54.426+0000] {standard_task_runner.py:60} INFO - Started process 3192 to run task
[2024-03-03T03:37:54.431+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'fetch_data_files', 'get_files_2', 'scheduled__2016-08-22T00:00:00+00:00', '--job-id', '1411', '--raw', '--subdir', 'DAGS_FOLDER/dummy_dag.py', '--cfg-path', '/tmp/tmpp6z29ojr']
[2024-03-03T03:37:54.437+0000] {standard_task_runner.py:88} INFO - Job 1411: Subtask get_files_2
[2024-03-03T03:37:54.473+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.11/site-packages/***/settings.py:194 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2024-03-03T03:37:54.547+0000] {task_command.py:423} INFO - Running <TaskInstance: fetch_data_files.get_files_2 scheduled__2016-08-22T00:00:00+00:00 [running]> on host 716a53fa82e1
[2024-03-03T03:37:54.766+0000] {taskinstance.py:2510} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='fetch_data_files' AIRFLOW_CTX_TASK_ID='get_files_2' AIRFLOW_CTX_EXECUTION_DATE='2016-08-22T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2016-08-22T00:00:00+00:00'
[2024-03-03T03:37:54.776+0000] {logging_mixin.py:188} INFO - [1922, 1923, 1924, 1925, 1926, 1927, 1928, 1929, 1930, 1931, 1932, 1933, 1934, 1935, 1936, 1937, 1938, 1939, 1940, 1941, 1942]
[2024-03-03T03:37:54.777+0000] {python.py:202} INFO - Done. Returned value was: None
[2024-03-03T03:37:54.796+0000] {taskinstance.py:1149} INFO - Marking task as SUCCESS. dag_id=fetch_data_files, task_id=get_files_2, execution_date=20160822T000000, start_date=20240303T033754, end_date=20240303T033754
[2024-03-03T03:37:54.886+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2024-03-03T03:37:54.918+0000] {taskinstance.py:3309} INFO - 0 downstream tasks scheduled from follow-on schedule check
